### PR TITLE
Site change

### PR DESCRIPTION
**Pull request by 5c13nc3Guy**
New owner.

### CODE REVIEW

**Comment by 5c13nc3Guy** [comment]
the new title has been changed.

File: index.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
 <body>
   <div class="section">
     <div class="container w-container">
-      <h4 class="heading">Test site</h4>
-      <h6 class="heading-2">MediaFire</h6><img src="images/idf_logo.png" loading="lazy" width="233" srcset="images/idf_logo.png 500w, images/idf_logo.png 800w, images/idf_logo.png 903w" sizes="232.9861297607422px" alt=""></div>
+      <h4 class="heading">Pascals site</h4>
```
Comment (pasc-ed) [inline index.html:21]: Site must be in Uppercase

Comment (5c13nc3Guy) [reply index.html:21]: merci beucoup

Comment (5c13nc3Guy) [reply index.html:21]: no pascal, that is crazy talk!

**Review by pasc-ed — APPROVED** [review]
All good, ok for merge.

**Pull request merged**